apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -74,6 +74,9 @@ const displayAssistantMessage = (status: string, chatMessageText: string) => {
 };
 
 const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({message}) => {
+  const botTitle = useSelector(
+    (state: {lab: AichatState}) => state.lab.levelProperties?.botTitle
```

Comment:
Instead of creating a separate `AichatState` type, I think we should just keep this as `LabState` since that's what it is, and instead cast the levelproperties to `AichatLevelProperties`, like `(state.lab.levelProperties as AichatLevelProperties | undefined)?.botTitle`

Reply:
Gotcha - thank you!